### PR TITLE
Vanilla 1.8

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Upgraded the site to Vanilla 1.8

## QA

- ./run
- Check the site in-browser at http://0.0.0.0:8011/
- Compare it to the live site and see that it is the same or better

### CODE REVIEW

**Comment by kwm14** [comment]
## Design review
Looking good @deadlight just a small tweak on mobile viewport.

### /brand
- [ ]Icons/SVGs sitting on top of text when viewed on mobile

screenshot 2018-12-11 at 17 19 09

**Comment by deadlight** [comment]
@kwm14 updated

**Comment by kwm14** [comment]
Updates LGTM 👍

File: _sass/main.scss [hunk 1]
```diff
@@ -41,3 +41,8 @@ $color-navigation-background: $color-brand;
   text-indent: -10000px !important;
 }
 
+//XXX KW 2018-12-12 - temp fix for:
+// https://github.com/vanilla-framework/vanilla-framework/issues/2080
```
Comment (nottrobin) [inline _sass/main.scss]: :heart: very thorough

**Comment by nottrobin** [comment]
Spacing here looks worse than it used to:

New:

image

Old:

image

**Comment by nottrobin** [comment]
Although it's not a regression, you might consider removing the padding on the breadcrumbs on `/brand` as a fly-by:

image

**Pull request merged**